README.md

```diff
@@ -1,196 +1,134 @@
-
-Arguments are mostly used to skip build unneeded binaries. For example, you want switch to prebuild binary folder `bin/dir_01`:
-```commandline
-> python tools/build.py -libs -mx-classes -bin-dir bin/dir_01
-```
-### Debugging
-#### Visual Studio 2019
-Recommended software for debugging, has really nice mixed python and C stack debugging. Provides to you ability of interactive code evaluation. You can make breakpoints move step by step, watch variables and etc.
-
-##### 1. Run Blender with the Add-on
-Make sure you have no installed addon for Blender version you want to use; remove installed version if needed.
-
-```commandline
-> set BLENDER_EXE="C:\Program Files\Blender Foundation\Blender 2.93\blender.exe"
-> python tools\run_blender_test_addon.py --window-geometry 0 0 1920 1080
-```
-
-##### 2. Attach Visual Studio to process
-Press menu Debug -> Attach to Process... or use hotkey`Ctrl+Alt+P`. In opened window choose Blender process, now you connected and allowed to debug.
-Also use build-in Python debugger in realtime. Turn on with `Debug -> Windows -> Python Debug Interactive. 
-
-#### Blender
- The easiest way to [build Blender](https://wiki.blender.org/wiki/Building_Blender/Windows) in Release or RelWithDebInfo and add `#pragma optimize( "", off )`.
-
-#### PyCharm
-```python
-import pydevd
-pydevd.settrace('localhost', port=52128, stdoutToServer=True, stderrToServer=True, suspend=False)
-```
-
-### Logging
-Using python's `logging` module underneath, hdusd.utils.logging has functions similar to logging. It also includes callable class Log which provides simplified interface to do logging.
-
-##### Code example:
-```python
-from hdusd.utils import logging
-log = logging.Log(tag='export.mesh')
-
-log("sync", mesh, obj)
-```
-For example, `logging.debug(*argv, tag)` where argv is what is printed (same as with print) and tag is string suffix for logger name, for filtering, so that `logging.limit_log(name, level_show_always)` will allow to filter out what doesn't start with `name`(expect levels equal or above `level_show_always`).
-
-Configure your session `configdev.py` (loaded very early) can be used to include code like `limit_log`.
-
-##### configdev.py example:
-```python
-from .utils import logging
-logging.limit_log('default', logging.DEBUG) # set debug level
-
-from . import config
-config.engine_use_preview = False # turn off preview of materials
-```
+# Blender USD Hydra Addon
+
+With Pixar's USD system emerging as a powerful tool for 3D graphics pipelines and interchange. This addon uses AMD Radeon™ ProRender renderer in USD Hydra rendering system in Blender.
+
+## Additional Documentation
+- [Pixar USD](https://graphics.pixar.com/usd/docs/index.html)
+- [Hydra](https://graphics.pixar.com/usd/docs/USD-Glossary.html#USDGlossary-Hydra)
+- [MaterialX](http://www.materialx.org/)
+
+## Requirements
+Currently, this addon works only with [Blender 4.0+](https://www.blender.org/download/) in Windows and Linux.

```

Comment:
Why can't we use Mac?

Reply:
MacOS is not supported yet, we have a task for it.